dot/network/service_test.go

```diff
@@ -60,11 +60,11 @@ func createServiceHelper(t *testing.T, num int) []*Service {
 func createTestBlockResponseMessage(t *testing.T) *BlockResponseMessage {
 	t.Helper()
 
+	const blockRequestSize = 128
 	msg := &BlockResponseMessage{
-		BlockData: []*types.BlockData{},
+		BlockData: make([]*types.BlockData, blockRequestSize),
```

Comment:
You need to change the code below to not `append` to the slice but use the index instead